cmd/rule-evaluator/main.go

```diff
@@ -569,6 +581,34 @@ func sendAlerts(s *notifier.Manager, externalURL *url.URL) rules.NotifyFunc {
 	}
 }
 
+func googleCloudLink(projectID, expr string, endTime, startTime time.Time) string {
+	// Clone URL to avoid mutating the original.
+	url := googleCloudBaseURL
+	url.Path += ";endTime=" + endTime.Format(time.RFC3339)
+	url.Path += ";startTime=" + startTime.Format(time.RFC3339)
+
+	// These settings reflect the default on metrics explorer for majority of use-cases.
```

Comment:
suggestion: Add a comment about URL format.

Is there a reference doc we can link here to inform the URL format and parameters? Or did you reverse-engineer this from a raw URL? Even that's the case, giving an example in a comment would be helpful.

Reply:
Sadly, it was reverse-engineered.

Reply:
cc @yqlu in case he knows of any docs around the ME URLs